index.html

```diff
@@ -3,9 +3,31 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta main="keyword " content="html, Code the Dream">
+    <meta name="author" content="TomN">
     <title>TomN's Portfolio</title>
 </head>
 <body>
-    
+    <h1>Tom Namhay</h1>
+    <section id="About">
+        <h2>About</h2>
```

Comment:
I think they do want you to actually add content here, but it sounds like you know that and are working toward that. If you need help getting an image to show up, I'm glad to help also!